src/app/ssh-keys/ssh-key-list/ssh-key-list.component.ts

```diff
@@ -40,7 +40,7 @@ export class SshKeyListComponent {
   public selectSshKeyPair(sshKeyPair: SSHKeyPair): void {
     this.router.navigate(['view', sshKeyPair.name], {
       relativeTo: this.route,
-      queryParamsHandling: 'preserve'
+      queryParams: {account: sshKeyPair.account}
```

Comment:
Please, add queryParamsHandling: 'preserve'

Reply:
if queryParams and queryParamsHandling: 'preserve' are used together, the query parameters are not adding to the routerLink

Reply:
@HeyRoach , is there any other way to pass params to route with queryParamsHandling: 'preserve'?